update our Code of Conduct to align with other UW eScience programs (#175)
uwhackweek/jupyterbook-template: book/CoC.md

# Participant Conduct

> The UW Data Science for Social Good program is run by the UW eScience Institute and, thus, all DSSG program participants are expected to comply with the following [Code of Conduct](https://escience.washington.edu/about/code-of-conduct/). Project teams are intentionally diverse across many categories including academic standing, domain field, technical experience, etc., and all program participants are expected to respect the diverse perspectives represented. Please also see the [eScience Equity Statement](https://escience.washington.edu/about/equity-statement/).

Reviewer: ```suggestion
We require all participants in the University of Washington (UW) eScience Hackweek program to comply with the [eScience Code of Conduct](https://escience.washington.edu/about/code-of-conduct/) copied below.

Hackweek participants are intentionally diverse across many categories including academic standing, domain field, technical experience, etc., and we expect everyone to respect all perspectives represented. Please also read the [eScience Equity Statement](https://escience.washington.edu/about/equity-statement/).
```

> # Code of Conduct

Reviewer: ```suggestion
## Code of Conduct
```

* Violent threats or language directed against another person
* Sexual language or images in public spaces
* Deliberate intimidation, stalking, or following
* Continuing to initiate interaction (including photography or recording) after being asked to stop
* Publication of private communication without consent
* Sustained disruption of talks, events, or communications
* Insults or put downs
* Nonconsensual or unwelcome physical contact
* Advocating for, or encouraging, any of the above behavior

Participants asked to stop any harassing behavior are expected to comply immediately. This applies to any eScience events or activities, either online or in-person. Participants violating these rules may be sanctioned or expelled from participation at the discretion of eScience staff. The organizers may take action to redress anything designed to, or with the clear impact of, disrupting the event or making the environment hostile for any participants. 

> Harassment and other code of conduct violations reduce the value of our programming for everyone. If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. You can make a report either personally or anonymously. To personally report, contact the eScience executive director, Sarah Stone, at sstone3@uw.edu or another trusted staff member. Anonymous reports can be made [here](bit.ly/uwhackweekfeedback). If you are experiencing a problem or issue that needs to be addressed by someone outside the program, you may contact the UW Office of the Ombud at ombuds@uw.edu or 206-543-6028.

Reviewer: ```suggestion
Harassment and other code of conduct violations reduce the value of our programming for everyone. If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. You can make a report either personally or anonymously. To personally report, contact the eScience executive director, Sarah Stone, at sstone3@uw.edu or another trusted staff member. Anonymous reports can be made [here](https://bit.ly/uwhackweekfeedback). If you are experiencing a problem or issue that needs to be addressed by someone outside the program, you may contact the UW Office of the Ombud at ombuds@uw.edu or 206-543-6028.
```

Reviewer: replace email or add `escience-hackweeks@uw.edu`?

[SafeCampus](https://www.washington.edu/safecampus/) (206-685-7233) is also available at any time to anonymously discuss safety and well-being concerns for yourself and others. SafeCampus is the University of Washington’s violence-prevention and response program, providing resources to UW students, staff, faculty and community members. For more information, visit their [What to Expect](https://www.washington.edu/safecampus/what-to-expect) page.

Thank you for helping to make our institute inclusive, welcoming and safe.

> [Contact us here.](bit.ly/uwhackweekfeedback)

Reviewer: ```suggestion
[Contact us here.](https://bit.ly/uwhackweekfeedback)
```
